integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/setup/standalone/CcWithDefaultConf.java

```diff
@@ -62,13 +62,13 @@ void start() throws Exception {
         ApictlUtils.createProject( "prod_openAPI.yaml", "prod_petstore");
         ApictlUtils.createProject( "sand_openAPI.yaml", "sand_petstore");
         ApictlUtils.createProject( "security_openAPI.yaml", "custom_authheader_petstore");
-        ApictlUtils.createProject( "vhost1_openAPI.yaml", "vhost1_petstore", null, "vhost1_deploy_env.yaml", null);
-        ApictlUtils.createProject( "vhost2_openAPI.yaml", "vhost2_petstore", null, "vhost2_deploy_env.yaml", null);
+        ApictlUtils.createProject( "vhost1_openAPI.yaml", "vhost1_petstore", null, "vhost1_deploy_env.yaml", null,null);
+        ApictlUtils.createProject( "vhost2_openAPI.yaml", "vhost2_petstore", null, "vhost2_deploy_env.yaml", null,null);
```

Comment:
```suggestion
        ApictlUtils.createProject( "vhost1_openAPI.yaml", "vhost1_petstore", null, "vhost1_deploy_env.yaml", null, null);
        ApictlUtils.createProject( "vhost2_openAPI.yaml", "vhost2_petstore", null, "vhost2_deploy_env.yaml", null ,null);
```
There are multiple places with the same suggestion